server/datastore/mysql/scheduled_queries.go

```diff
@@ -37,7 +37,7 @@ func (ds *Datastore) ListScheduledQueriesInPackWithStats(ctx context.Context, id
 			JSON_EXTRACT(ag.json_value, '$.system_time_p95') as system_time_p95,
 			JSON_EXTRACT(ag.json_value, '$.total_executions') as total_executions
 		FROM scheduled_queries sq
-		JOIN queries q ON (sq.query_name = q.name)
+		JOIN (SELECT * FROM queries WHERE team_id IS NULL) q ON (sq.query_name = q.name)
```

Comment:
Hmm there's no available index on `queries.name` anymore since the index is now:

```
UNIQUE KEY `idx_team_id_name_unq` (`team_id_char`,`name`),
```

I think if that unique key had `name` first, it could use the index.

Reply:
The uniqueness constraint was updated because now queries must be unique on a per team basis. I think that can be rewritten as `INNER JOIN queries ON team_id_char = '' AND sq.query_name = q.name`